From 7facec455d825d317caffb8cafc4e4fec60065d9 Mon Sep 17 00:00:00 2001
From: monsta <monsta@inbox.ru>
Date: Wed, 16 Aug 2017 16:44:59 +0300
Subject: [PATCH] don't abort on unknown DBus signal name

it's external data, so it's not very clever to put assert on it

same as done in:
https://github.com/mate-desktop/mate-panel/commit/40b31b2117f8a1776d2f23d97c60547c3c5a70e4
---
 src/gpm-kbd-backlight.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gpm-kbd-backlight.c b/src/gpm-kbd-backlight.c
index ae8e0fa..c646855 100644
--- a/src/gpm-kbd-backlight.c
+++ b/src/gpm-kbd-backlight.c
@@ -326,7 +326,7 @@ gpm_kbd_backlight_on_dbus_signal (GDBusProxy *proxy,
        return;
    }
 
-   g_assert_not_reached ();
+   g_debug ("signal '%s' not handled!", signal_name);
 }
 
 /**
-- 
2.13.5
